app/controllers/cars_controller.rb

```diff
@@ -6,24 +6,31 @@ class CarsController < ApplicationController
 
   # GET /cars or /cars.json
   def index
-    @cars = Car.where(user: current_user)
+    @cars = Car.where(user: current_user).order("created_at ASC")
   end
 
   # GET /cars/1 or /cars/1.json
   def show
+    index
```

Comment:
I used that because of the button cars that I implemented in navbar and following the logic to show only cars that user have the button don't show in new car path and in edit car path. Is it bad practice

Reply:
I can't imagine not using helpers anymore, that's why they call them so 😊